torch/csrc/jit/passes/onnx/eval_peephole.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <torch/csrc/jit/ir/ir.h>
+
+namespace torch {
+namespace jit {
+
+void EvalPeepholeONNX(Block* b, std::map<std::string, IValue>& paramDict);
```

Comment:
why call it eval peephole? do we have plan to add more optimizations?

Reply:
if in future we might add more passes so that's why we use more generic name